### ISSUE TITLE
ability to add sections name for ingredients [and/or] steps in a single recipe

### ISSUE THREAD

**Issue by andryzas** [Recipe-Hero/Recipe-Hero#133]
sometimes recipes have couple sections of ingredients (e.g. to make a dough and toppings) or separate steps descriptions

INGREDIENTS:

1 dough
1.1 flour
1.2 water

2 toppings
2.1 2 cups of sugar
2.2 200 grams of sour cream

STEPS:

1 make dough:
1.1 step one
1.2 step two

2 make toppings:
2.1 step three
2.2 step four

3 finalize:
3.1 final step one
3.2 final step two

it would be nice to have one more button/field next to 'add new ingredient/step' something like 'add section name'.,

in current version it's impossible to distinct ingredients for different parts of recipe. with steps I can manually add html tag within the step description, so it's not that big issue yet it would be a nice-to-have feature and more user friendly approach.

**Comment by bryceadams** [comment]
Yep, noticed this before. Hopefully will figure out a nice way to do this soon :)
